### PR TITLE
Fix BTS-191

### PR DESCRIPTION
**Pull request by danielhlarkin**
### Scope & Purpose

*(Please describe the changes in this PR for reviewers)*

- [x] :hankey: Bugfix 
- [ ] :pizza: New feature 
- [ ] :hammer: Refactoring 
- [ ] :book: CHANGELOG entry made
- [ ] :muscle: The behavior in this PR was *manually tested*
- [ ] :computer: The behavior change can be verified via automatic tests

#### Backports:

- [ ] No backports required
- [x] Backports required for: 3.7

#### Related Information

*(Please reference tickets / specification etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] GitHub issue / Jira ticket number: BTS-191
- [ ] Design document: 

### Testing & Verification

Jenkins: https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/12085/

### CODE REVIEW

File: arangod/Transaction/Manager.cpp [hunk 1]
```diff
@@ -649,16 +652,16 @@ Result Manager::statusChangeWithTimeout(TransactionId tid, transaction::Status s
...
 
-Result Manager::abortManagedTrx(TransactionId tid) {
-  return statusChangeWithTimeout(tid, transaction::Status::ABORTED);
+Result Manager::abortManagedTrx(TransactionId tid, std::string const& database) {
+  return statusChangeWithTimeout(tid, database, transaction::Status::ABORTED);
 }
 
 Result Manager::updateTransaction(TransactionId tid, transaction::Status status,
-                                  bool clearServers) {
+                                  bool clearServers, std::string database) {
```
Comment (jsteemann) [inline arangod/Transaction/Manager.cpp:664]:
```suggestion
                                  bool clearServers, std::string const& database) {
```

File: arangod/Transaction/Manager.h [hunk 2]
```diff
@@ -196,14 +196,16 @@ class Manager final {
...
+                                 transaction::Status status);
 
   /// @brief hashes the transaction id into a bucket
   inline size_t getBucket(TransactionId tid) const {
     return std::hash<TransactionId>()(tid) % numBuckets;
   }
 
-  Result updateTransaction(TransactionId tid, transaction::Status status, bool clearServers);
+  Result updateTransaction(TransactionId tid, transaction::Status status, bool clearServers,
+                           std::string database = "" /* leave empty to operate across all databases */);
```
Comment (jsteemann) [inline arangod/Transaction/Manager.h:208]:
```suggestion
                           std::string const& database = "" /* leave empty to operate across all databases */);
```

**Review by jsteemann — COMMENTED** [review]
Code LGTM to me, but as this is a behavior change, we should have a CHANGELOG entry, and add tests.

**Comment by danielhlarkin** [comment]
https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/12146/

**Review by jsteemann — APPROVED** [review]

**Pull request merged**